response.proto

```diff
@@ -3,11 +3,20 @@ import "type.proto";
 
 package pbnavitia;
 
+enum PathType {
+    Road = 0;
+    SharedCycleWay = 1;    // Shared use lane (could be shared with pedestrians)
+    DedicatedCycleWay = 2; // Dedicated cycle lane
+    SeparatedCycleWay = 3; // A separate cycle lane (physical separation from the main carriageway
+    Residential = 4;
+}
```

Comment:
Will there be an "OppositeCycleWay" to model rue de Charenton for example?

Reply:
Valhalla doesn't have this information yet. Maybe someday...